src/Perpetuum/Perpetuum.csproj

```diff
@@ -1207,7 +1213,6 @@
     <Folder Include="Properties\" />
     <Folder Include="Services\ProductionEngine\Methods\" />
     <Folder Include="Services\Relay\RequestHandlers\" />
-    <Folder Include="Comparers\" />
```

Comment:
Where has this folder gone. iirc it container the equality comparer for the timeout player object.

Reply:
Well this is still in the file:
```
    <Compile Include="Comparers\EntityComparer.cs" />
    <Compile Include="EntityFramework\IEntity.cs" />
```
but I am not sure why the folder include is removed...